virttest/utils_net.py

```diff
@@ -1129,7 +1129,7 @@ def wrap_init(*args, **kargs):
                     raise Exception("Check of OpenVSwitch failed.")
             except Exception, e:
                 logging.debug("Host does not support OpenVSwitch: %s", e)
-
+                raise exceptions.TestFail("Host does not support OpenVSwitch")
```

Comment:
I can remove logging.debug message. If that is ok.
But not failing test at this point will lead test to ERROR out with below message which is misleading:

04:43:08 ERROR| AttributeError: 'NoneType' object has no attribute 'list_br'
04:43:08 ERROR|